vtex/faststore
# docs: OrderPlaced integration

## What's the purpose of this pull request?


This PR adds documentation on how to enable the latest OrderPlaced in FastStore stores.

## How does it work?

The markdown file is rendered in HTML using Docusaurus and will be live at faststore.dev. The following image is how this is being rendered (when running locally).

![Screen Shot 2022-01-28 at 10 43 17](https://user-images.githubusercontent.com/27777263/151557130-686fb062-3407-4867-afd0-fd25b2ebd260.png)

## How to test it?

Check the `netlify/faststore-docs/deploy-preview` deploy preview attached to this PR.
